<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamepreparation</class>
 <widget class="QDialog" name="gamepreparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Start game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:  url(:/images/gamefon.png);
background-repeat: no-repeat;
text-transform: uppercase; 
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_game">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/arrow-right.png);
background-repeat: no-repeat;
border: 0;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="plr1_rchange">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image: url(:/images/arrow-r.png);background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>71</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/player3.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="plr1_lchange">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png);background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/player4.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="plr2_lchange">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png);background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="plr2_rchange">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>410</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image: url(:/images/arrow-r.png);background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gamer1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);

color: rgb(253, 177, 167);
border: 2px solid #fff; 
color: #fff;
background-color: rgb(255, 176, 167);

padding-left: 30px;

border-radius:10%;
</string>
   </property>
   <property name="text">
    <string>Гравець 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gamer2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>520</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(253, 177, 167);
border: 2px solid #fff; 
color: #fff;
background-color: rgb(255, 176, 167);
padding-left: 30px;
border-radius:10%;</string>
   </property>
   <property name="text">
    <string>Гравець 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="ball">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recources.qrc">:/images/ball.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="ball_rchange">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image: url(:/images/arrow-r.png);background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ball_lchange">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png);background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(233, 160, 157); 

background-color: #fff;


  opacity: 0.4;
border-radius:10%;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="location">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>550</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(253, 177, 167);
border: 2px solid #fff; 
color: #fff;
background-color: rgb(255, 176, 167);
padding-left: 30px;
border-radius:10%;</string>
   </property>
   <property name="text">
    <string>Локація</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-l.png);
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-image:url(:/images/arrow-r.png);
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="place">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recources.qrc">:/images/ship33.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
